src/main/java/com/google/cloud/spanner/pgadapter/statements/MatcherStatement.java

```diff
@@ -1,66 +0,0 @@
-// Copyright 2020 Google LLC
```

Comment:
No longer needed, this is now handled automatically for both the simple and extended query protocol.